webapp/tests/Unit/Controller/API/ClarificationControllerTest.php

```diff
@@ -132,12 +133,17 @@ public function testAddInvalidData(string $user, array $dataToSend, string $expe
             $url .= '/' . $dataToSend['id'];
         }
         $data = $this->verifyApiJsonResponse($method, $url, 400, $user, $dataToSend);
-        static::assertEquals($expectedMessage, $data['message']);
+        if (str_starts_with($expectedMessage, '/')) {
+            static::assertMatchesRegularExpression($expectedMessage, $data['message']);
+        } else {
+            static::assertEquals($expectedMessage, $data['message']);
+        }
     }
 
     public function provideAddInvalidData(): Generator
     {
-        yield ['demo', [], "Argument 'text' is mandatory."];
```

Comment:
We don't display any text anymore?

Reply:
When you submit *nothing* there is an empty 400 (or 422 if we change the above thing) since Symfony doesn't know what to do. As soon as you supply *any* field (even an unknown one) we do display an error. It's a bit annoying but IMHO not a very big deal.

This test though should be added with an empty message probably.